READMEs/developers.md

```diff
@@ -132,3 +132,20 @@ Specifically, [follow this workflow](https://docs.oceanprotocol.com/concepts/con
 Release for pip etc.
 
 Specifically, [follow the Release Process instructions](../RELEASE_PROCESS.md).
+
+
+## Bonus: Test code quality
+
+We use Codacy to test for quality of code. Here's how.
+
+Export Codacy project token. If needed, [generate your own](https://docs.codacy.com/repositories-configure/integrations/project-api/)
+```console
+export CODACY_PROJECT_TOKEN=2f4be1e422ca43b19c506ec97bebd9eb
```

Comment:
Is this the "real" value? Is it suppposed to be public? In the Github Action, we are encrypting it. I always thought it was supposed to be a secret, but I might be wrong.

Reply:
Yep, likely supposed to be private. That's why this PR is still in "draft" mode. By the time the final PR goes through, it won't be there. Just trying to have something to work against while I figure this out.